adagrams/game.py

```diff
@@ -1,11 +1,143 @@
+import random
+
+letter_pool = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
 def draw_letters():
-    pass
+    """
+    function to create a random hand of 10 letters
+    no input
+    output is a list of 10 random letters
+    """ 
+    # assign variable names to the letters and max usage num 
+    letter_list, usage_list = list(letter_pool.keys()), list(letter_pool.values())
+    
+    hand = []
+    while len(hand) < 10:
+        position = random.randint(0, len(letter_list)-1)
+        letter_frequency = usage_list[position] # assigns the random int to a usage number
+        random_letter = letter_list[position] # assigns the random int to a letter
+        
+        if hand.count(random_letter) < letter_frequency:
+            add_letter = hand.append(random_letter)
+        else:
+            continue
+    return hand
```

Comment:
Right now your code isn't actually utilizing the weight distribution of the letters. Currently, all letters have the same chance of being selected (they all can only appear their respective limits). `randint` is only randomly selecting from a pool of integers from `0` to `26`. This means that `Z` is just as likely as likely to be selected as `E`. You would need it to select from a pool of integers from `0` to `98` to have the correct distribution.  How could we change your letter pool to consider this weighted distribution?